diff --git a/node_modules/gatsby-source-pg/gatsby-node.js b/node_modules/gatsby-source-pg/gatsby-node.js
index 3475fa3..eef5b02 100644
--- a/node_modules/gatsby-source-pg/gatsby-node.js
+++ b/node_modules/gatsby-source-pg/gatsby-node.js
@@ -17,6 +17,7 @@ exports.createSchemaCustomization = async (props, options) => {
   const pool = new Pool({
     connectionTimeoutMillis: 30 * 1000,
     connectionString,
+    max: 1
   });
 
   const graphqlSchema = await createSchema(pool, postgresSchema, rest);
