diff --git a/node_modules/react-hubspot-form/dist/index.js b/node_modules/react-hubspot-form/dist/index.js
index 7f1ffa5..563d5af 100644
--- a/node_modules/react-hubspot-form/dist/index.js
+++ b/node_modules/react-hubspot-form/dist/index.js
@@ -61,7 +61,7 @@ var HubspotForm = function (_React$Component) {
 					target: '#' + this.el.getAttribute('id'),
 					onFormSubmit: function onFormSubmit($form) {
 						// ref: https://developers.hubspot.com/docs/methods/forms/advanced_form_options
-						var formData = $form.serializeArray();
+						var formData = $form.serializeArray ? $form.serializeArray() : $form.dataset;
 						_this2.props.onSubmit(formData);
 					}
 				});
@@ -82,6 +82,7 @@ var HubspotForm = function (_React$Component) {
 				_this3.createForm();
 				_this3.findFormElement();
 			};
+			script['data-osano'] ="ESSENTIAL";
 			script.src = '//js.hsforms.net/forms/v2.js';
 			document.head.appendChild(script);
 		}
